EasyPost.Tests/ShipmentTest.cs

```diff
@@ -223,6 +223,8 @@ public async Task TestCreateTaxIdentifiers()
             Assert.AreEqual("IOSS", shipment.tax_identifiers[0].tax_id_type);
         }
 
+        [Ignore]
+        // test is for some reason failing to pull a proper recording when playing back. Only test doing this
```

Comment:
Please investigate, it's important we fix this if possible (feel free to create a follow up task just for this, drop in the backlog.)

Reply:
Will do, running into the same issue with the Java VCR, so it's probably something with how the VCR is matching the request on replay.

Reply:
I wonder if the duplicate address creation requests (we make a To address and From address, but they have the same exact data) is interfering.